packages/ui/src/Toast.module.scss

```diff
@@ -8,9 +8,12 @@
   max-width: c.$toastMaxWidth;
   padding: c.$grid * 1.5 c.$grid * 2.5;
 
+  .icon {
+    margin-right: c.$grid * 2.5;
+  }
+
   .content {
     word-break: break-all;
-    margin-left: c.$grid * 2.5;
   }
```

Comment:
Is there any difference?

Reply:
None. I honestly don't remember the reasoning behind this. Can roll back, if you prefer to include it in `.content`.

Reply:
I asked just out of curiosity. No need to rollback :)